cpp/src/arrow/flight/server.h

```diff
@@ -184,6 +184,10 @@ class ARROW_FLIGHT_EXPORT FlightServerBase {
   /// domain socket).
   int port() const;
 
+  /// \brief Get the address that the Flight server is listening on.
```

Comment:
Should this return a `std::vector<Location>` instead, in case the server is listening on several addresses?

Reply:
We don't really model that capability; you can add multiple ports by hooking into gRPC yourself, but the server APIs expect one location to listen on.